src/TiciTacaToeyGameEngine.ts

```diff
@@ -11,12 +11,26 @@ import {
   COMPLETED_GAME_STATUS,
   CalculateWinnerInputType,
   CalculateWinnerOutputType,
+  Player,
+  Timer,
 } from "./model";
 import WebSocket = require("ws");
 import uniq from "lodash.uniq";
 
 const EMPTY_POSITION = "-";
+let timePerPlayer = 5000;
```

Comment:
This is also incorrect because when the first game updates the value of timePerPlayer to 10000 all subsequent games for all other players will end up using this value.